api/config/config_local.json

```diff
@@ -54,7 +54,10 @@
     "operationalTerraWorkspaceName": "aouwgscohortextraction",
     "extractionMethodConfigurationNamespace": "aouwgscohortextraction",
     "extractionMethodConfigurationName": "HelloWorld",
-    "extractionMethodConfigurationVersion": 2
+    "extractionMethodConfigurationVersion": 34,
+    "extractionCohortsDataset": "fc-aou-cdr-synth-test.wgs_extraction_cohorts",
```

Comment:
I'm wondering whether this belongs here, or if it's better as an extension to `cdr_config_*.json`